pages/docs/v3/updating/plugins/3-0.md

```diff
@@ -84,6 +122,10 @@ We believe `resolve()` and `reject()` better reflect the Promise-like flow inten
 
 Previously, calling `resolve()` with no arguments resulted in an empty object being sent to the JavaScript layer. Because this is unlike the behavior of JavaScript's `Promise.resolve()`, as of Capacitor 3, `undefined` is sent instead.
 
+### Saving calls
+
+The `save()` method has deprecated and a replacement `keepAlive` property has been added. The recommended patterns for saving a call have been documented to clarify the behavior. [Read more about that here](/docs/core-apis/saving-calls).
```

Comment:
This link will not work until https://github.com/ionic-team/capacitor-site/pull/238 is merged.